robot_skills/src/robot_skills/hero.py

```diff
@@ -40,7 +40,6 @@ def __init__(self, wait_services=False):
        
         #rename joint names
         self.parts['leftArm'].joint_names = self.parts['leftArm'].load_param('skills/arm/joint_names')
```

Comment:
Shouldn't this also be generic?

Reply:
Jep, but it isn't at the moment. So please make an issue for it.